README.md

```diff
@@ -21,34 +21,19 @@ finding something going wrong. There's a long list of known issues
 Check out [the wiki page](https://github.com/new-xkit/XKit/wiki/Writing-a-New-Extension)!
 
 ## Develop XKit
-Download [Node.js](https://nodejs.org/download/) for your platform.
+Review the project prerequisites and learn how to build XKit from source [over on the wiki](/wiki/Build-XKit).
```

Comment:
I could be wrong, but when viewed from https://github.com/new-xkit/XKit then this will redirect to https://github.com/new-xkit/XKit/blob/master/wiki/Build-XKit. Would it be a better idea to use an absolute URL instead?